templates/text.jsx

```diff
@@ -0,0 +1,7 @@
+import { templates } from 'core/js/reactHelpers';
+
+export default function(view, model) {
```

Comment:
You'll need switch the arguments around so that it is `(model, view)` now.

`instruction` and `mobileInstruction` should now do their thing automatically.

Reply:
`instruction` and `mobileInstruction` swap is working well for me